jarbas/README.md

```diff
@@ -67,7 +67,7 @@ All these endpoints accepts any combination of the following parameters:
 * `year`
 * `order_by`: `issue_date` (default) or `probability` (both descending)
 * `in_latest_dataset` (_boolean_, `1` parses to `True`, `0` to `False`)
-
+* `congressperson_name__contains` (contains)
```

Comment:
This became outdated, but probably (I mean, if you agree with my feedbacl) it wil become a generic `serch` field. It would be awesome if we document all fields included in this search ; )